### PR TITLE
LPS-204019 Repeatable fields are not saved for editor configuration client extensions

### PR DESCRIPTION
**Pull request by markocikos**
### References

- [LPS-204019 Repeatable fields are not saved for editor configuration client extensions](https://issues.liferay.com/browse/LPS-204019)

### What is the goal of this PR?

A bugfix. We are treating repeatable fields as strings in backend.

cc @boton

### What does it look like?

https://github.com/liferay-frontend/liferay-portal/assets/5435511/95f79f3c-864a-4bda-858e-7420fc525f7d

### Steps to reproduce

See JIRA ticket

### CODE REVIEW

**Comment by markocikos** [comment]
ci:test:relevant

File: modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/EditorConfigContributorCETImplFactoryImpl.java [hunk 1]
```diff
@@ -50,11 +52,19 @@ public UnicodeProperties getUnicodeProperties(
...
+			"editorNames",
+			StringUtil.merge(
+				ParamUtil.getStringValues(portletRequest, "editorNames"),
+				StringPool.NEW_LINE)
 		).put(
-			"portletNames", ParamUtil.getString(portletRequest, "portletNames")
+			"portletNames",
+			StringUtil.merge(
+				ParamUtil.getStringValues(portletRequest, "portletNames"),
+				StringPool.NEW_LINE)
```
Comment (dsanz) [inline modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/EditorConfigContributorCETImplFactoryImpl.java:67]: Nice catch :tada:, we lost those bits at some point from the PoC 

@juanjofgliferay @boton this is the thing we were looking for this morning. No more actions on your side

**Review by dsanz — APPROVED** [review]
LGTM ;)

**Comment by dsanz** [comment]
ci:forward

**Comment by markocikos** [comment]
The failure in https://github.com/liferay-frontend/liferay-portal/pull/3908#issuecomment-1865039701 looks unrelated to me.

**Comment by markocikos** [comment]
ci:forward:force

**Pull request closed without merging**